### ISSUE TITLE
Test case import do not work if file has only one line

### ISSUE THREAD

**Issue by moimael** [msjit/testcasedb#101]
We tried to import test case via xls and found out that if the xls file contains only the title row and one data row, import fail. As soon as we add a second data row, import works.

**Comment by joffems** [comment]
@moimael I will investigate. Is your single line entry a complete line? Is every column filled in?

**Comment by moimael** [comment]
@joffems Yes the line is complete and well formed, with no special characters.
